apps/web/src/pages/admin/seats/index.tsx

```
@@ -0,0 +1,27 @@
+import ErrorBoundary from "@src/components/commons/ErrorBoundary";
+import Error from "@src/pages/seats/Error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)